### PR TITLE
`Development`: Update server dependencies

### PR DESCRIPTION
**Pull request by krusche**
This PR updates server dependencies.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,13 +21,13 @@ plugins {
...
     id "com.github.node-gradle.node" version "3.4.0"
-    id "com.diffplug.spotless" version "6.9.1"
+    id "com.diffplug.spotless" version "6.10.0"
     // this allows us to find outdated dependencies via ./gradlew dependencyUpdates
     id "com.github.ben-manes.versions" version "0.42.0"
     id "com.github.andygoossens.modernizer" version "1.6.2"
     id "com.gorylenko.gradle-git-properties" version "2.4.1"
-    id "info.solidsoft.pitest" version "1.7.4"
-    id "org.owasp.dependencycheck" version "7.1.1"
+    id "info.solidsoft.pitest" version "1.9.0"
```
Comment (b-fein) [inline build.gradle:29]: This update also requires changing updating line 395 to 
`junit5PluginVersion = "1.0.0"`.

Comment (krusche) [reply build.gradle:29]: thanks, done

**Review by stefanwaldhauser — APPROVED** [review]
☑️ Looked through changed dependencies changelogs and deployed on testserver.

**Review by ge65cer — COMMENTED** [review]
LGTM.

**Pull request merged**